hack/namespace_scoped.yaml

```
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: acme
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: test-account
+  namespace: acme
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: test-account
+  namespace: acme
+rules:
+- apiGroups: ['', 'extensions', 'apps']
+  resources: ['*']
+  verbs: ['*']
+- apiGroups: ['batch']
+  resources:
+  - jobs
+  - cronjobs
+  verbs: ['*']
+- apiGroups:
+    - bpfman.io
+  resources:
+    - bpfnsprograms
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - bpfman.io
+  resources:
+    - bpfnsprograms/status
+  verbs:
+    - get
+- apiGroups:
+    - bpfman.io
+  # resources: ['xdpnsprograms']
+  resources: ['bpfnsapplications', 'tcnsprograms', 'tcxnsprograms', 'uprobensprograms', 'xdpnsprograms']
+  verbs:
+    - create
+    - delete
+    - get
+    - list
```

This is what fixed it:

```suggestion
    - list
    - watch
```